src/backends.js

```diff
@@ -14,12 +14,34 @@
 
 import { GerritBackend } from './backends/gerrit';
 import { RietveldBackend } from './backends/rietveld';
+import * as state from './state';
 
+/**
+ * Connects to the backend server to fetch the changes.
+ * @record
+ */
+// eslint-disable-next-line no-unused-vars
```

Comment:
How about making this class a real base class of two backends? Suppressing no-unused-vars sounds not very good.